createdisk.sh

```diff
@@ -184,6 +210,14 @@ copy_additional_files $1 $libvirtDestDir
 
 tar cJSf $libvirtDestDir.tar.xz $libvirtDestDir
 
+# HyperKit image generation
+# This must be done after the generation of libvirt image as it resuse some of
+# the content of $libvirtDestDir
```

Comment:
Given that the libvirt and hyperkit disk images are going to be the same, we could use the same bundle for both. When used with libvirt, it will contain a few unneeded  files (the kernel and initramfs), but that will not make a significant difference in disk usage.

Reply:
@cfergeau it is not only the extra files but the metadata content also, I prefer to have separate tarball for separate hypervisor.

Reply:
The extra metadata can be ignored too. Main advantage of a single tarball is less download time if you use both macos and linux.

Reply:
@cfergeau so you want to say have only single libvirt tarball which going to work with linux and mac with hyperkit?

Reply:
yes, I think this is what I would have done, less tarballs to generate, to download, ... But if we are going to ship crc+bundle as a single "binary", then it's really not important whether the libvirt and hyperkit bundles are the same or are different.